src/loaders/js.ts

```diff
@@ -19,15 +19,15 @@ export const jsLoader: Loader = async (input, { options }) => {
   let contents = await input.getContents();
 
   // declaration
-  if (options.declaration && !input.srcPath?.match(DECLARATION_RE)) {
```

Comment:
**Note**: Unnecessary because it's already checked here:

https://github.com/unjs/mkdist/blob/69ed1480c872cb9bc084afbacb254b168abeef4b/src/loaders/js.ts#L12